### DISCUSSION TITLE
Android port

### DISCUSSION THREAD

**Discussion by HanzoDev1375** [manticore-projects/jsqlformatter#18] [Ideas]
hi can you proting code in Android?

**Comment by manticore-projects** [comment]
Greetings!

Of course it can be done, but why? What would be a relevant use-case for JSQLFormatter on Android?

**Reply by HanzoDev1375** [reply]
Hello and don't worry, I am an Android developer, the reason we say porting to Android is for my app, you can check my repository 

My app name is spirit web actually it's a mobile ide I add sql language to my app it just needs a code arranger your repository is great for this I'll be glad to release a pr or atleast android compatible version
